__tests__/app.test.js

```diff
@@ -0,0 +1,55 @@
+const db = require('../db/connection');
+const seed = require('../db/seeds/seed');
+const testData = require('../db/data/test-data/index');
+
+const request = require('supertest');
+const app = require('../db/app');
+const topics = require('../db/data/test-data/topics');
+
+beforeEach(()=> {
+    return seed(testData)
+});
+afterAll(()=> {
+    return db.end()
+});
+
+describe('GET /api/topics', () =>{
+    test('status:200, responds with an array of topics', () =>{
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then(({ body: { topics } }) => {
+            expect(topics).toHaveLength(3);
+            topics.forEach((topic) => {
+                expect(topic).toMatchObject({
+                    slug: expect.any(String),
+                    description: expect.any(String),
+                });
+            });
+        });
+    });
+});
+
+describe('GET /api/articles', () =>{
+    test('status:200, responds with an array of articles and relevant properties', () =>{
```

Comment:
Great test, could you add in an assertion to check that the articles are sorted by date in ascending order please.

(jest-sorted from the notes might help you with this)